ReactAndroid/src/main/java/com/facebook/react/uimanager/layoutanimation/LayoutAnimationController.java

```diff
@@ -66,6 +68,12 @@ public void reset() {
   }
 
   public boolean shouldAnimateLayout(View viewToAnimate) {
+    if (viewToAnimate instanceof ReactViewGroup) {
```

Comment:
Not sure about this approach but layout animation didn't work properly at all here since the new position of the view isn't docked yet it gets animated in a weird way. What I did is simply add a flag to disable layout animation on ReactViewGroup.